_data/sw_libraries.yml

```diff
@@ -80,3 +80,9 @@
           - userhost-in-names
           - batch
           - sasl
+    - name: BlueIRC
+      # ref: https://github.com/MicroDroid/blueirc/blob/development/src/tk/microdroid/blueirc/Handler.java#L59
+      v3.1:
```

Comment:
This should be under the `support:` key, similar to other libraries (see [this](https://github.com/MicroDroid/ircv3.github.io/blob/master/_data/sw_libraries.yml#L15) as an example). Otherwise, this looks good!